src/goto-programs/json_goto_trace.cpp

```diff
@@ -44,27 +45,59 @@ void convert_assert(json_objectt &json_failure,
 }
 
 void convert_decl(json_objectt &json_assignment,
-  conversion_dependenciest &conversion_dependencies)
+  conversion_dependenciest &conversion_dependencies,
+  const trace_optionst &trace_options)
 {
   const goto_trace_stept& step=conversion_dependencies.step;
-  jsont &location=conversion_dependencies.location;
+  jsont &json_location=conversion_dependencies.location;
   const namespacet &ns=conversion_dependencies.ns;
 
   irep_idt identifier=step.lhs_object.get_identifier();
 
   json_assignment["stepType"]=json_stringt("assignment");
 
-  if(!location.is_null())
-    json_assignment["sourceLocation"]=location;
+  if(!json_location.is_null())
```

Comment:
(comment for entire commit) These changes should be squashed with the previous commit.

Reply:
I suggest to improve the commit message.